cartridges/custom_cartridge_1/cartridge/experience/components/commerce_assets/banner.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+var Template = require('dw/util/Template');
+var HashMap = require('dw/util/HashMap');
+/**
+ * Render logic for the component.
+ */
+module.exports.render = function (context) {
+   var model = new HashMap();
+   model.bannerMessage = context.content.alt;
+ return new Template('experience/components/commerce_assets/banner').render(model).text;
```

Comment:
Watch out for the indentations and formatting.

Reply:
Fixed